manifests/metallb-frr.yaml

```diff
@@ -484,6 +487,11 @@ spec:
               mountPath: /var/run/frr
             - name: frr-conf
               mountPath: /etc/frr
+          # The command is FRR's default entrypoint & waiting for the log file to appear and tailing it.
+          # If the log file isn't created in 60 seconds the tail fails and the container is restarted.
+          # This workaround is needed to have the frr logs as part of kubectl logs -c frr < speaker_pod_name >.
+          command: ["/bin/sh"]
+          args: ["-c","/sbin/tini -- /usr/lib/frr/docker-start & attempts=0; until [[ -f /etc/frr/frr.log || $attempts -eq 60 ]]; do sleep 1 && attempts=$(( $attempts + 1 )); done && tail -f /etc/frr/frr.log"]
```

Comment:
Just one ask. Can you make this multiline? I think it would be easier to read / understand.

Reply:
changed, is it better now?